backend/domain/item.go

```diff
@@ -21,7 +21,36 @@ type Item struct {
 	UpdatedAt   string
 }
 
+type GetItemResponse struct {
+	ID           int64      `json:"id"`
+	Name         string     `json:"name"`
+	CategoryID   int64      `json:"category_id"`
+	CategoryName string     `json:"category_name"`
+	UserID       int64      `json:"user_id"`
+	Price        int64      `json:"price"`
+	Description  string     `json:"description"`
+	Status       ItemStatus `json:"status"`
+}
+
 type Category struct {
 	ID   int64
 	Name string
 }
+
+func ConvertToGetItemResponse(items []Item, categoryName string) []GetItemResponse {
```

Comment:
```suggestion
func(i *Item) ConvertToGetItemResponse() GetItemResponse
```

[imo]
こういうのを想定してた！
このままだと、異なるcategoryNameを持つItemsが来たときに正しいCategoryNameが設定できないと思う

Reply:
> 異なるcategoryNameを持つItemsが来たときに正しいCategoryNameが設定できないと思う

言われてみたらそうだった！！！直しておきます